tensorboard/plugins/histogram/README.md

```diff
@@ -239,4 +239,4 @@ bumps over the integer values rather than perfect spikes.
 
 ### All Together Now
 Finally, we can concatenate all of the data into one funny-looking curve.
-![](https://www.tensorflow.org/images/tensorboard/histogram_dashboard/11_all_combined.png)
\ No newline at end of file
+![](https://www.tensorflow.org/images/tensorboard/histogram_dashboard/11_all_combined.png)
```

Comment:
It looks like I fixed a missing newline at EOF. (Well, my editor did automatically.)
`````diff
$ git show 005e2dac08 tensorboard/plugins/histogram/README.md | sed -n '/^diff/,$p'
diff --git a/tensorboard/plugins/histogram/README.md b/tensorboard/plugins/histogram/README.md
index 3d753b5..543e102 100644
--- a/tensorboard/plugins/histogram/README.md
+++ b/tensorboard/plugins/histogram/README.md
@@ -46,7 +46,7 @@ Once that code runs, we can load the data into TensorBoard via the command line:
 
 
 ```sh
-tensorboard --logdir=/tmp/histogram_example
+tensorboard --logdir /tmp/histogram_example
 ```
 
 Once TensorBoard is running, load it in Chrome or Firefox and navigate to the
@@ -239,4 +239,4 @@ bumps over the integer values rather than perfect spikes.
 
 ### All Together Now
 Finally, we can concatenate all of the data into one funny-looking curve.
-![](https://www.tensorflow.org/images/tensorboard/histogram_dashboard/11_all_combined.png)
\ No newline at end of file
+![](https://www.tensorflow.org/images/tensorboard/histogram_dashboard/11_all_combined.png)
`````